ckanext/switzerland/helpers/validators.py

```diff
@@ -442,29 +439,37 @@ def ogdch_validate_formfield_temporals(field, schema):
     This validator is only used for form validation.
     The data is extracted form the temporals form fields and transformed
     into a form that is expected for database storage:
-    "temporals": [{"start_date": 0123456789, "end_date": 0123456789}]
+    "temporals": [{"start_date": <date as isodate>,
+    "end_date": <date as isodate>}]
     """
     def validator(key, data, errors, context):
```

Comment:
do we really need  `errors,` and `context` as an argument here ?

Reply:
I am not sure about that, but all the validators have that signature, so I want to leave it as such for now. Would that be okay for you?

Reply:
I would remove arguments that we do not use in the function

Reply:
These arguments are required for the functions returned by the validators, which are all called with the same arguments at a separate place in the code. Please don't remove them.

Reply:
Hmm yes just found that out the hard way debugging for quite some time ... should have read your comment @bellisk Thanks a lot for this comment. But I found it only after I had this idea finally myself this could be the issue, why the temporals suddenly had stopped to work.